packages/cli/generators/relation/index.js

```diff
@@ -0,0 +1,442 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const _ = require('lodash');
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const debug = require('../../lib/debug')('relation-generator');
+const inspect = require('util').inspect;
+const path = require('path');
+const chalk = require('chalk');
+const utils = require('../../lib/utils');
+const tsquery = require('../../lib/ast-helper');
+const ast = require('ts-simple-ast');
+const relationUtils = require('./relationutils');
+
+const RelationBelongsTo = require('./relationBelongsTo');
+const RelationHasMany = require('./relationHasMany');
+const RelationHasOne = require('./relationHasOne');
+
+const ERROR_INCORRECT_RELATION_TYPE = 'Incorrect Relation Type';
+const ERROR_MODEL_DOES_NOT_EXIST = 'model does not exist.';
+const ERROR_NO_DESTINATION_MODEL_SELECTED = 'No destination model selected';
+const ERROR_NO_MODELS_FOUND = 'No models found in';
+const ERROR_NO_SOURCE_MODEL_SELECTED = 'No source model selected';
+const ERROR_RELATION_TYPE_PARAMETER_SHOULD_BE_SPECIFIED =
+  "'relationType' parameter should be specified.";
+const ERROR_SOURCE_MODEL_PRIMARY_KEY_DOES_NOT_EXIST =
+  'Source model primary key does not exist.';
+
+const PROMPT_BASE_RELATION_CLASS = 'Please select the relation type';
+const PROMPT_MESSAGE_SOURCE_MODEL = 'Please select source model';
+const PROMPT_MESSAGE_TARGET_MODEL = 'Please select target model';
+const PROMPT_MESSAGE_PROPERTY_NAME = 'Property name for the relation';
```

Comment:
I find this prompt little bit confusing. How about the following?

```
Source property name for the relation getter 
```

Reply:
Fixed.